environment.yml

```diff
@@ -3,31 +3,6 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - python=3
-  - numpy>=1.14
-  - sympy>=1.4
-  - cgen
-  - scipy
-  - matplotlib
-  - pytest
-  - flake8>=2.1.0
-  - parso>=0.1.0
-  - nbval
-  - cached-property
-  - psutil>=5.1.0
-  - sphinx
-  - sphinx_rtd_theme
-  - distributed>=1.27
-  - cython
-  - click
-  - codecov
-  - pytest-cov
-  - multidict
-  - frozendict
-  - nb_conda_kernels
```

Comment:
this doesn't appear in requirements.txt ?

the following also are missing (not sure why):
- cython
- sphinx_rtd_theme ( I think necessary for rendering of documentation?)
- sphinx
- parso>=0.1.0 ??

Reply:
Gonna assume old unused dependency because everything works on all os/env and platforms